File: 20140212/fizzbuzz.rb
def toStr i
  if i % 15 == 0
    "FizzBuzz"
  elsif i % 3 == 0
    "Fizz"
  elsif i % 5 == 0
    "Buzz"
  else
    i.to_s
  end
end

def fizzBuzz array
  array.map{|i| toStr i}
end

puts fizzBuzz (1..100)

File: 20140222/fizzbuzz.py
def toStr(i):
    if i % 15 == 0:
        return "FizzBuzz"
    elif i % 3 == 0:
        return "Fizz"
    elif i % 5 == 0:
        return "Buzz"
    else:
        return i

def fizzBuzz(array):
    for i in array:
        print(toStr(i))

fizzBuzz(range(1,100))
